modules/node_modules/@ncigdc/containers/RepositoryPage.js

```diff
@@ -120,40 +133,31 @@ export const RepositoryPageQuery = {
     files_size: null,
     files_sort: null,
     filters: null,
-    idAutocompleteCases: null,
-    runAutocompleteCases: false,
-    idAutocompleteFiles: null,
-    runAutocompleteFiles: false,
+    autocompleteTypes: [],
+    idAutocomplete: null,
+    runAutocomplete: false,
   },
   fragments: {
-    /*
     autocomplete: () => Relay.QL`
-      fragment on Repository {
-        cases {
-          hits(first: 5, id_autocomplete: $idAutocompleteCases) {
-            edges @include(if: $runAutocompleteCases) {
-              node {
-                case_id
-                project {
-                  project_id
-                }
+      fragment on Root {
+        query (query: $idAutocomplete types: $autocompleteTypes) @include(if: $runAutocomplete) {
```

Comment:
can't seem to, graphql throws
`[GraphQLError('Argument "if" has invalid value "1".\nExpected type "Boolean", found "1".',)]` and casting it throws
```
141Uncaught Error: Relay transform error 
Syntax Error RepositoryPage (2:79) Unexpected !

1: fragment RepositoryPage_AutocompleteRelayQL on Root {
2:         query (query: $idAutocomplete types: $autocompleteTypes) @include(if: !!$idAutocomplete) {
                                                                                 ^
3:           hits {
```